app/src/main/res/values/strings.xml

```diff
@@ -700,6 +700,14 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="create_new_note_hint">Best write the note in the locally spoken language or otherwise in English.</string>
     <string name="quest_cyclewayPartSurface_title">What\'s the surface of the cycleway here?</string>
     <string name="quest_footwayPartSurface_title">What\'s the surface of the footway here?</string>
+    <string name="quest_board_type_history">History</string>
+    <string name="quest_board_type_geology">Geology</string>
+    <string name="quest_board_type_plants">Plants</string>
+    <string name="quest_board_type_wildlife">Wildlife</string>
+    <string name="quest_board_type_nature">Nature (multiple topics)</string>
+    <string name="quest_board_type_notice_board">It is a notice board</string>
+    <string name="quest_board_type_public_transport">Public transport</string>
```

Comment:
What about adding sight as "Sight (viewpoint or panorama)" to better explain it. There are quite a few of them around.

Reply:
We'll leave out this option for now because "sight" is a generic option that often might be specified in more detail. If the user is forced to leave a note in case of a sight that does not fit into these categories, it will help to establish other specific values.

Reply:
Okay, it seems to be the preferred option compared to panorama for example. I just find notes a hassle to have to then come back and manually tag them on the OSM website.